<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="r10_settings_groupbox_2">
         <property name="title">
          <string>GSPro</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPlainTextEdit" name="ipaddress_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::PreventContextMenu</enum>
            </property>
            <property name="toolTip">
             <string>GSPro IP Address</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>GSPro IP Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="gspro_window_name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>GSPro Window Title</string>
            </property>
            <property name="placeholderText">
             <string>GSPro Window Title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="port_edit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>GSPro Port</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>GSPro Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="gspro_api_window_name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>GSPro API Window Title</string>
            </property>
            <property name="placeholderText">
             <string>GSPro API Window Title</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPlainTextEdit" name="gspro_path_edit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>31</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Application Path(Opyional)</string>
              </property>
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::WidgetWidth</enum>
              </property>
              <property name="placeholderText">
               <string>GSPro Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="file_browse_button">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Launch Monitor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="launch_monitor_combo">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="r10_settings_groupbox">
         <property name="title">
          <string>R10 BT Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPlainTextEdit" name="r10_settings_device_name_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Device Name</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Device Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="r10_settings_humidity_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Humidity (percent (0 - 1)</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Humidity (percent (0 - 1)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="r10_settings_temperature_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Temperature (fahrenheit )</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Temperature (fahrenheit )</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="r10_settings_altitude_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Altitude (feet)</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Altitude (feet)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="r10_settings_airdensity_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Air Density (kg/m^3)</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Air Density (kg/m^3)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="r10_settings_tee_distance_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Tee Distance to R10 (feet)</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Tee Distance to R10 (feet)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Relay Server Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPlainTextEdit" name="relay_server_ip_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Relay Server IP Address</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Relay Server IP Address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="relay_server_port_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>31</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Relay Server Port</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Relay Server Port</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>MEVO+ Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="auto_start_all_label_2">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string extracomment="GSPro Spin Axis Correction"/>
              </property>
              <property name="text">
               <string>Offline Mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mevo_offline_mode_combo">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string extracomment="GSPro Spin Axis Correction"/>
              </property>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New Item</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Default OCR Device </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="default_device_combo">
               <property name="minimumSize">
                <size>
                 <width>130</width>
                 <height>0</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>New Item</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="auto_start_all_label">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Auto Start All Apps</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="auto_start_all_apps_combo">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>New Item</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="save_button">
           <property name="toolTip">
            <string>New Device</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close_button">
           <property name="toolTip">
            <string>New Device</string>
           </property>
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
